[2025-10-06T21:32:06.815+0000] {processor.py:161} INFO - Started process (PID=43) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:06.816+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:32:06.818+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:06.817+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:06.826+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:06.825+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:32:06.826+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:06.846+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.041 seconds
[2025-10-06T21:32:37.185+0000] {processor.py:161} INFO - Started process (PID=52) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:37.185+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:32:37.190+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:37.190+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:37.201+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:37.200+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:32:37.201+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:32:37.224+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.051 seconds
[2025-10-06T21:33:07.622+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:07.623+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:33:07.625+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:07.625+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:07.632+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:07.631+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:33:07.633+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:07.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.047 seconds
[2025-10-06T21:33:38.001+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:38.003+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:33:38.006+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:38.006+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:38.012+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:38.011+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:33:38.013+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:33:38.052+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.060 seconds
[2025-10-06T21:34:08.413+0000] {processor.py:161} INFO - Started process (PID=82) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:08.414+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:34:08.416+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:08.416+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:08.425+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:08.424+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:34:08.425+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:08.445+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.040 seconds
[2025-10-06T21:34:38.807+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:38.812+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:34:38.821+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:38.821+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:38.840+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:38.839+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:34:38.841+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:34:38.863+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.082 seconds
[2025-10-06T21:35:09.261+0000] {processor.py:161} INFO - Started process (PID=102) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:09.261+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:35:09.263+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:09.263+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:09.268+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:09.268+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:35:09.271+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:09.292+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.039 seconds
[2025-10-06T21:35:39.664+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:39.666+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:35:39.667+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:39.667+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:39.674+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:39.672+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:35:39.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:35:39.697+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.040 seconds
[2025-10-06T21:36:10.261+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:10.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:36:10.270+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:10.269+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:10.284+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:10.283+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:36:10.286+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:10.310+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.081 seconds
[2025-10-06T21:36:40.815+0000] {processor.py:161} INFO - Started process (PID=132) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:40.817+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:36:40.821+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:40.821+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:40.830+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:40.830+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:36:40.831+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:36:40.851+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.051 seconds
[2025-10-06T21:37:11.304+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/Flask_API.py
[2025-10-06T21:37:11.305+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Flask_API.py for tasks to queue
[2025-10-06T21:37:11.308+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:11.308+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:37:11.321+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:11.319+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/Flask_API.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/Flask_API.py", line 9, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:37:11.321+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Flask_API.py
[2025-10-06T21:37:11.348+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Flask_API.py took 0.059 seconds
